email_tracking/app.rb

```diff
@@ -0,0 +1,62 @@
+require 'sinatra/base'
+require 'easypost'
+require 'sendgrid-ruby'
+require 'tilt/erb'
+require 'pry'
+require 'dotenv'
+
+include SendGrid
```

Comment:
Is this necessary. Can you just reference things as `SendGrid::Email` instead? This `include` adds to `Kernel` which is normally bad practice. I'm not sure why SendGrid advocates doing this.